tests/integration_test/services_test.py

```
@@ -304,6 +321,661 @@
+        assert from_date.month == now.month
+        assert from_date.year == now.year
+
+        last_day = calendar.monthrange(now.year, now.month)[1]
+        assert to_date.day == last_day
+
+
+
+
+
+
+import calendar
```

Ruff (0.12.2)

Redefinition of unused `calendar` from line 326

Remove definition: `calendar`

(F811)